### ISSUE TITLE
Test gateways against omnipay

### ISSUE THREAD

**Issue by barryvdh** [thephpleague/omnipay-common#109]
We should enfore better testing with Omnipay Common.
1 thing we can do is test both dev, stable en lowest versions, like this:
https://github.com/thephpleague/omnipay-paypal/pull/137

Second thing is that we might want to trigger the testcase for all gateways somehow. Not sure if that is even possible? Preferably _before_ a PR is merged, but that would mean pulling down all gateways and running all tests..
Any ideas?

**Comment by delatbabel** [comment]
In the case of an individual gateway, I normally ask the submitter of a PR to test this in production before I tag for release.  I'm not sure what else we can do -- these issues fail due to a combination of:

* The tests pass, but the code fails due to gateway issues
* Some gateways don't always make it easy to get hold of test accounts.
* Some gateways behave differently between test and production accounts.

**Comment by barryvdh** [comment]
Yes but I mean specifically for omnipay common,  as that is used across all gateways. A breaking change here can affect a gateway we are not aware of. So after each change,  all the gateway tests should be triggered also.

**Comment by delatbabel** [comment]
If you're asking for wave-a-magic-wand fixes to make that happen, sorry but I don't have any.  :)  I guess we just have to be careful with changes to omnipay-common. It was good that this got caught on the dev branch before it got tagged for release though.
